csaps/_shortcut.py

```diff
@@ -1,219 +1,230 @@
-# -*- coding: utf-8 -*-
```

Comment:
Somehow all lines in the file have been changed. Could you make changes only for new added/deleted lines?